openmetadata-ui/src/main/resources/ui/src/rest/userAPI.ts

```diff
@@ -235,7 +235,7 @@ export const updateUserAccessToken = async ({
       JWTTokenExpiry?: string;
       tokenName?: string;
     },
-    AxiosResponse<PersonalAccessToken[]>
+    AxiosResponse<PersonalAccessToken>
```

Comment:
@Ashish8689 , can you please explain this change?

Reply:
so create `personalAccessToken` for `PUT` request returns us the `PersonalAccessToken ` and not the array of it.
<img width="936" alt="image" src="https://github.com/open-metadata/OpenMetadata/assets/66266464/76926028-7574-4809-8f27-a8cc76018a55">
<img width="936" alt="image" src="https://github.com/open-metadata/OpenMetadata/assets/66266464/539ede8b-97fe-4b23-b4f4-2b12cf6b7bc8">